### PR TITLE
Check if lang key exists before applying name masking

### PR DESCRIPTION
**Pull request by KameiB**
Raw Ores and Raw Ore Blocks now first check if its lang key exists and use it, before trying to use the custom name masking method (which works only for English tho. In other languages it turns into "Raw Iron Ingot", etc.).  
Added lang keys for Iron and Gold, so modpack creators have examples and create their own lang keys in the resources folder.

### CODE REVIEW

**Comment by SmileycorpMC** [comment]
It's a good idea, but it would require way too much set up on the part of modpack creators, so I don't think it works completely.

That being said translation is a big issue that I didn't even think about, I think adding a dictionary for translated versions of Ingot or Dust in various languages may be good here, so instead of looking for litterally "Ingot" it will check the lang file for the word to remove.

**Comment by KameiB** [comment]
Yeah, localization support is not an easy task.
Sadly, I don't think just removing the equivalent of "Ingot" would work well in other languages.
For instance, "Iron Ingot" in Spanish is "Lingote de hierro", where:
Lingote = Ingot
de = of
hierro = iron

In Japanese, the word "Iron" is written as "**鉄**", but 
Iron ore = **鉄**鉱石 (Iron ore)
Raw iron = **鉄**の原石 (Rough stone of iron)
Iron ingot = **鉄**インゴット (Iron ingot)

TL;DR
For language support, is better to just create the individual english lang keys.
For other languages, the community can make language packs for their favorite modpack.
With that, modpack creators don't need to worry about manually adding all lang files in the modpack.

The PR I made makes it so, if the lang key exists, show its contents as the item name. If not, use your method.
That way, modpack creators don't need to worry about languages, but language pack creators have the opportunity to correctly translate your items in their language.
